portfolio/src/main/webapp/index.html

```diff
@@ -25,6 +25,27 @@ <h1 id="title">My Portfolio</h1>
       <div id="greeting-container"></div>
     
     <img src="/images/city-birdeye.jpg" id="img-container"/>
+
+    <form action="/form-handler" method="POST">
+        <p>Enter some text here:</p>
+        <textarea name="text-input">hello world</textarea>
+        <br/>
+        <input type="submit" />
+    </form>
+
+    <form action="/contact" method="POST">
+        <p>Name:</p>
+        <textarea name="input-name">Your Name Here</textarea>
+        <p>Email:</p>
+        <textarea name="input-email">Your Email Here</textarea>
+        <p>Phone Number</p>
+        <textarea name="input-number">Your Phone Number Here</textarea>
+        <p>Got Beef? Leave it here.</p>
+        <textarea name="input-jokes">Got jokes or beef, leave it here</textarea>
+        <br/>
+        <input type="submit" />
+    </form>
```

Comment:
Is there supposed to be a ContactHandler for this? If so, it wasn't added.

Reply:
What exactly are you referring to? Something from one of the earlier teachme tutorials? I tried looking for something related to the form like what your talking about but couldnt find what your talking about